Bugsnag/Helpers/BSGUtils.h

```diff
@@ -28,3 +28,7 @@ NSString *_Nullable BSGStringFromThermalState(NSProcessInfoThermalState thermalS
 NS_ASSUME_NONNULL_END
 
 __END_DECLS
+
+static inline NSString * _Nullable bsg_string_from_class(Class _Nullable cls) {
+    return cls ? NSStringFromClass((Class _Nonnull)cls) : nil;
+}
```

Comment:
* Function declarations should be inside the `__BEGIN_DECLS` and `NS_ASSUME_NONNULL_BEGIN` "section"
* Functions should should follow the naming convention for this header; e.g. `BSGStringFromClass`